elements/sdk-platform.bst

```diff
@@ -0,0 +1,263 @@
+kind: stack
+
+depends:
+
+# There are certain elements inside gnome-sdk and freedesktop-sdk that we need
+# to build ourselves. As a result, we can't depend on gnome-sdk directly. This
+# list should be kept in sync with the dependencies in sdk-platform.bst from
+# gnome-sdk.
```

Comment:
I think I'd add a link to the file in gnome-build-meta, for easier maintenance?

That would also make it clear which version of gnome-build-meta this is based upon.

I guess this is the file? https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/gnome-3-34/elements/sdk-platform.bst

Reply:
Good idea! Thank you :)